### PR TITLE
Minor UI improvements

### PR DESCRIPTION
**Pull request by mamchenkov**
* Better display of the related record links (using labels)
* Fix mistakenly escaped HTML in the fields of the associated records
* Add plus icon on the 'Add' button of the index top menu

### CODE REVIEW

File: src/Events/ViewMenuListener.php [hunk 1]
```diff
@@ -53,9 +53,10 @@ public function getIndexMenuTop(Event $event, Request $request, array $options)
         $urlAdd = ['plugin' => $request->plugin, 'controller' => $request->controller, 'action' => 'add'];
 
         $btnAdd = $appView->Html->link(
+            '<span class="glyphicon glyphicon-plus"></span> ' .
```
Comment (flangofas) [inline src/Events/ViewMenuListener.php]: @mamchenkov 
If you prefer:

```php
$view->Html->icon('plus')
```

Comment (mamchenkov) [inline src/Events/ViewMenuListener.php]: @flangofas I don't see the `icon()` method on the `HtmlHelper` ... where is it coming from?

Comment (mamchenkov) [inline src/Events/ViewMenuListener.php]: @flangofas sounds useful, but doesn't work.  The `$appView` variable doesn't seem to have the BootstrapUI helper loaded.  We'll need to look into it in a separate PR.

Thanks for the pointer!

Comment (flangofas) [inline src/Events/ViewMenuListener.php]: @mamchenkov done. #189

Comment (mamchenkov) [inline src/Events/ViewMenuListener.php]: @flangofas Thanks! Awesome! :+1:

**Pull request merged**